devtools/test_dashboard/devtools.js

```diff
@@ -5,6 +5,7 @@
 var Fuse = require('fuse.js/dist/fuse.common.js');
 var mocks = require('../../build/test_dashboard_mocks.json');
 var credentials = require('../../build/credentials.json');
+var constants = require('@src/plots/geo/constants');
```

Comment:
don't you want `tasks/util/constants`?

Reply:
This is exactly the same as what is used in `geo_test`
https://github.com/plotly/plotly.js/blob/69ffb2c4a3fa6ed88d9cafd9db497741d8f24c23/test/jasmine/tests/geo_test.js#L6

Reply:
You are right. We are not really using local files here and in `geo_test` on this branch.

Reply:
Good catch! 
Fixed in https://github.com/plotly/plotly.js/pull/5492/commits/343faa898cdc92e1e5ceee821c369bb09a135c4a.